lib/pedicel/ec.rb

```diff
@@ -53,21 +63,22 @@ def symmetric_key(shared_secret: nil, private_key: nil, merchant_id: nil, certif
     def shared_secret(private_key:)
       begin
         privkey = OpenSSL::PKey::EC.new(private_key)
-      rescue => e
+      rescue StandardError => e
         raise EcKeyError,
               "invalid PEM format of private key for EC: #{e.message}"
       end
 
       begin
         pubkey = OpenSSL::PKey::EC.new(ephemeral_public_key).public_key
-      rescue => e
-        raise EcKeyError, "invalid format of ephemeralPublicKey (from token) for EC: #{e.message}"
+      rescue StandardError => e
+        raise EcKeyError,
+              "invalid format of ephemeralPublicKey (from token) for EC: #{e.message}"
```

Comment:
Is rubocop asking for this formatting?

Reply:
Yes, both regarding line-lengths and `StandardError`.
I would probably prefer omitting `StandardError`, so I'd be happy to change it back :)

Reply:
Would changing it back be okay?